app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command, file_name, new_file_name = command.split()
```

Comment:
this line can cause an error, I think it should be swapped with your check

Reply:
I don't know what to do. Please advise.

Reply:
> this line can cause an error, I think it should be swapped with your check

I don't know what to do. Please advise.